content/en/containers/amazon_ecs/apm.md

```diff
@@ -261,8 +300,24 @@ env[DD_VERSION] = $DD_VERSION
 
 {{< /programming-lang-wrapper >}}
 
+##### IMDSv2
```

Comment:
I think this section should be bundled closer with the "task definition" example. Right now it's
- task definition example
- Ruby code example
- task definition example
I think both examples that involve entrypoint modifications should be closer together.

Reply:
This IMDSv2 sample is actually outside all the code blocks, so it will show up at the bottom for each. But I can update this to make it a little more clear that this is the generic one for all.

It didn't feel necessary / looked to wordy to have both the IMDSv1 and IMDSv2 sample in each code block.

Reply:
If all the tracers can support `"export DD_AGENT_HOST=$(curl http://169.254.169.254/latest/meta-data/local-ipv4); <command used to launch application container>"`, then maybe just bundle the IMDSv1 and IMDSv2 sample into one common section, then have only a few tables for the languages that support code modifications?

Reply:
That makes sense. So I guess I'm for the current structure where:
- Broken down by lang
- Launch time `DD_AGENT_HOST` variable example for each
    - For the most part referencing a generic `<Python Startup Command>` 
    - Additional notes where applicable per language, adding in extra links/notes for context
- Code snippets per applicable languages added

But I'm totally open to adding extra notes for each language. Relative to the current status of this:
- Is the IMDSv2 example still reading as too unclear?
- Should Java be updated to `<Java Startup Command>` and make notes for `-javaagent` and link to that section as appropriate for the actual command?
- Should Python more explicitly provide uwsgi and those links as an example?

Reply:
I think the IMDSv1 can be simplified to:
```
"export DD_AGENT_HOST=$(curl http://169.254.169.254/latest/meta-data/local-ipv4); <command used to launch application container>"
```

The IMDSv2 looks good to me. 
```
"export TOKEN=$(curl -X PUT \"http://169.254.169.254/latest/api/token\" -H \"X-aws-ec2-metadata-token-ttl-seconds: 21600\"); export DD_AGENT_HOST=$(curl -H \"X-aws-ec2-metadata-token: $TOKEN\" http://169.254.169.254/latest/meta-data/local-ipv4); <Startup Command>"
```
This will work for all tracers so we don't have to get into specifics of how to set up APM in the containers focused pages.

Reply:
I think I'm going to have to leave the IMDSv1 section as is. Tried a few different iterations of including the generic `entryPoint` command like: 
```json
"entryPoint": [
  "sh",
  "-c",
  "export DD_AGENT_HOST=$(curl http://169.254.169.254/latest/meta-data/local-ipv4); <Startup Command>"
]
```

Then going including some minor details per language. However, it just looked too off. While Node, Ruby, and Go are all pretty much the same, having it separated for the other 4 languages didn't flow well. So having the little snippet there for each, while a little redundant, does read a lot easier in my opinion.

I've included links to the APM Docs where applicable to avoid having these docs be too focused on the tracer component and instead be more ECS focused.